[Workers] standard pricing announcement (#12759)
cloudflare/cloudflare-docs: content/workers/platform/pricing.md

# Pricing

{{<Aside type="warning">}}

> All Workers and Pages Functions currently billed under the [Bundled and Unbound pricing models](/workers/platform/pricing/#deprecated-usage-models) will be billed under the [Standard model](/workers/platform/pricing/#example-pricing-standard-usage-model) beginning on March 1, 2024.

Reviewer: ```suggestion
**Workers Standard pricing is now available**
New, [simplified pricing](https://blog.cloudflare.com/workers-pricing-scale-to-zero/) is available for Workers under a new usage model, called Workers Standard. [Opt-in](https://dash.cloudflare.com/?to=/:account/workers/standard/opt-in) to Workers Standard pricing today.

You have until March 1, 2024 to switch to new pricing on your own, after which all of your Workers and Pages Functions currently under the [Bundled and Unbound pricing models](/workers/platform/pricing/#deprecated-usage-models) will be automatically migrated to the [Standard model](/workers/platform/pricing/#example-pricing-standard-usage-model) for users on the Workers Paid plan. 
```

Reviewer: Moved the note under "Workers" up here to consolidate. Can we delete that note below (can't do it with Github suggestions).

# Pricing

> {{<Aside type="warning" header="Workers Standard pricing is now available">}}

Reviewer 2: ```suggestion
{{<Aside type="note" header="Workers Standard pricing is now available">}}
```

Reviewer: Can we keep this as a warning. It's important and I don't want it getting lost with the other notes on the page.

New, [simplified pricing](https://blog.cloudflare.com/workers-pricing-scale-to-zero/) is available for Workers under a new usage model, called Workers Standard. [Opt-in](https://dash.cloudflare.com/?to=/:account/workers/standard/opt-in) to Workers Standard pricing today.

> You have until March 1, 2024 to switch to new pricing on your own, after which all of your Workers and Pages Functions currently under the [Bundled and Unbound pricing models](/workers/platform/pricing/#deprecated-usage-models) will be automatically migrated to the [Standard model](/workers/platform/pricing/#example-pricing-standard-usage-model) for users on the Workers Paid plan. 

Reviewer: ```suggestion
 For users on the Workers Paid plan: You **have until March 1, 2024 to switch to new pricing on your own**, after which all of your Workers and Pages Functions currently under the [Bundled and Unbound pricing models](/workers/platform/pricing/#deprecated-usage-models) will be **automatically migrated** to the [Standard model](/workers/platform/pricing/#example-pricing-standard-usage-model). 
```
